design-system/Avatar.svelte

```diff
@@ -56,6 +59,16 @@
       }
     }
   }
+
+  function blockieBg(urn: string) {
+    const seed = urn.toLowerCase();
+    const avatar = createIcon({
+      seed,
+      size: 8,
+      scale: 16,
+    });
+    return avatar.toDataURL();
+  }
```

Comment:
We don't loose any of the descriptiveness when it's even more concise:
```suggestion
  function blockieBg(urn: string) {
    return createIcon({
      seed: urn.toLowerCase(),
      size: 8,
      scale: 16,
    });
  }
```

Reply:
we don't do `toDataURL()` here. is this still gonna work?

Reply:
Ah, missed that. Adjusted the previous comment.